tests/test_misc.py

```diff
@@ -1,30 +1,31 @@
 #!/usr/local/bin/python3.8
 # coding=utf-8
+# pylint: disable=no-self-use, protected-access, too-few-public-methods
 from collections.abc import MutableMapping
 
 from lbz.misc import MultiDict, NestedDict, Singleton, error_catcher, get_logger
 
 
-def test_NestedDict():
-    a = NestedDict()
-    a["a"]["b"]["c"]["d"]["e"] = "z"
-    assert a == {"a": {"b": {"c": {"d": {"e": "z"}}}}}
+def test_nested_dict():
+    nest = NestedDict()
+    nest["a"]["b"]["c"]["d"]["e"] = "z"
+    assert nest == {"a": {"b": {"c": {"d": {"e": "z"}}}}}
 
 
-def test_Singleton():
-    class A(metaclass=Singleton):
+def test_singleton():
+    class AClass(metaclass=Singleton):
         pass
 
-    a = A()
-    b = A()
-    c = A()
-    assert a == b == c
-    assert Singleton._instances[A] == a
-    a._del()
-    assert Singleton._instances.get(A) is None
+    a_inst = AClass()
+    b_inst = AClass()
+    c_inst = AClass()
+    assert a_inst == b_inst == c_inst
```

Comment:
I think here, more proper would be to use `is` because it will check if those three instances are exactly the same object in memory 😉